nym-vpn-apple/Scripts/UpdateCore.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
```

Comment:
It's a good practice for bash script to setup global error handling

basic example:
```bash
# at the very top of the script

# exit immediatly on non 0 status returnerd by a command
set -e
# treat unset variables and parameters (other than the special parameters  `@` or `*`) as an error
set -u
# in pipelineor  compound commands, exit the pipeline at the 1st  encountered non 0 status
set -o pipefail
```

for more advanced error handling, like how to pseudo try-catch, take a look at https://github.com/nymtech/nym-vpn-client/blob/main/.pkg/linux/install

Reply:
Updated 👍

Reply:
Cool! Now you trap, you probably want to set `set -E` as well, as by default errors emitted from command substitutions and subshell are not trapped